packages/palette/src/elements/Input/tokens.ts

```diff
@@ -48,8 +83,25 @@ export const INPUT_STATES: Record<State, any> = {
     ::placeholder {
       color: ${themeGet("colors.black30")};
     }
+
+    &:not(:placeholder-shown) + label {
+      color: ${themeGet("colors.black30")};
+      transform: translateY(-150%);
```

Comment:
All of the animation styling in here look great (checked out [the review app](https://new-input.artsy.net/)) 👍 

I wonder if there's any way we might be able to improve the perf a bit tho. It _could_ just be my machine, but it feels like the frame rate is a bit slow (or rather, could be faster). I know translate Y pushes to the gpu, but 🤔 

@MrSltun , @dzucconi - thoughts? Maybe its just fine... not too sure. 


https://github.com/artsy/palette/assets/236943/2273bffb-d8dc-4d81-9915-dba848da7e26

Reply:
I'd be interested to know if the color is what's slowing you down; otherwise no.

Totally fine on my M2 with 32GB of ram though 😆

Reply:
Oh interesting, i just went back to the tab and its all ok! It was due to low battery on my computer. Looks perfect now. 

Ignore!

Reply:
So eager for this to launch ⭐